apps/fluent-tester/src/FluentTester/TestComponents/Common/MenuPicker.windows.tsx

```diff
@@ -0,0 +1,37 @@
+export * from './MenuPicker.desktop';
+import * as React from 'react';
+import { Button, Text, View, StyleSheet } from 'react-native';
+
+interface MenuPickerProps {
+  prompt?: string;
+  selected?: any;
+  onChange?: (value: any, index?: number) => void;
+  collection?: any[];
+  style?: any;
+}
+
+export const MenuPicker: React.FunctionComponent<MenuPickerProps> = (props: MenuPickerProps) => {
+  const { prompt, onChange, collection } = props;
+  return (
+    <View style={menuPickerStyles.container}>
+      <Text style={menuPickerStyles.prompt}>{prompt}</Text>
+      {collection.map((value, index) => (
+        <Button onPress={() => onChange(value, index)} key={index} title={value} />
+      ))}
+    </View>
+  );
+};
```

Comment:
Wouldn't this render all the menu items always?

Reply:
I think React has special cases for arrays of children such that it will make a comparison using the key prop

Reply:
I meant that this will just render a View, with a Text component inside of it and a Button for each menu item. There's no "click to open menu" kinda thing happening here unless I am not understanding it properly